src/ConnectedMode/UI/DeleteConnection/ProjectInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.ConnectedMode.UI.ManageBinding;
+using SonarLint.VisualStudio.ConnectedMode.UI.ProjectSelection;
+
+namespace SonarLint.VisualStudio.ConnectedMode.UI.DeleteConnection;
+
+public record ProjectInfo(ServerProject ServerProject, SolutionInfoModel SolutionInfo);
```

Comment:
Not a strong opinion about this record. I found it a bit confusing while checking the PR. It contains the server-side project and the local project association. Maybe it can be renamed to `ConnectedModeProject`.

Reply:
Your proposal sounds very good!